src/codecoverage/bot/code_coverage_bot/github.py

```diff
@@ -46,6 +47,13 @@ def mercurial_to_git():
             r = requests.get('{}/gecko-dev/rev/hg/{}'.format(self.hg_git_mapper, mercurial_commit))
 
             if not r.ok:
+                for email in secrets[secrets.EMAIL_ADDRESSES]:
+                    self.notify_service.email({
+                        'address': email,
+                        'subject': 'Commit missing on Git',
+                        'content': 'HG commit {} is missing on Git'.format(mercurial_commit),
```

Comment:
`Mercurial commit {} is missing on the mapper service`

Also, use an f-string instead of using format.

Reply:
Should I generate another pull request, resolving the issues?

Reply:
Oh, no. You just need to update this one.

Reply:
Also, how do we know what caused the failure of tests in TaskCluster?

Reply:
At the bottom of the PR, you can see a link to Taskcluster (it says "Details"). When you open that, you can see the build and its logs.